installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -94,7 +100,19 @@ resource "null_resource" "chef_provision_jenkins_server" {
       "cd ${var.chefDestDir}/chef-export && sudo chef-client -z"
     ]
   }
+}
+
+resource "null_resource" "configureJenkinsDocker" {
+  count = "${var.scenario2or3}"
+  depends_on = ["null_resource.preJenkinsConfiguration", "aws_elasticsearch_domain.elasticsearch_domain"]
+  // Chef Process inside docker
```

Comment:
As mentioned, use 1 `count` variable name `dockerizedJenkins` and follow the `if/else` pattern described in this article: https://blog.gruntwork.io/terraform-tips-tricks-loops-if-statements-and-gotchas-f739bbae55f9

Reply:
Done